### ISSUE TITLE
.png's not working

### ISSUE THREAD

**Issue by andrewsanchez** [syl20bnr/spacemacs#5396]
PNG warning: Application built with libpng-1.2.50 but running with 1.6.20

@NJBS suggested that I move libpng to a temporary location and rebuild Emacs. How do I find
 out where libpng-1.2.50 and libnpg1.6.20 are so that I can try and follow this suggestion?  I'm also not sure which one I should move.

```
truthling@Truthling:~$ whereis libpng
libpng: /usr/lib/libpng.la /usr/lib/libpng16.so /usr/lib/libpng16.la /usr/lib/libpng.so /usr/include/libpng /usr/share/man/man3/libpng.3 /usr/share/man/man3/libpng.3.gz
```

Are any of these the files I should move before rebuilding?

The script I used to build Emacs 24.5 is at https://github.com/andrewsanchez/build-emacs/blob/master/build-emacs.sh and I'm on Ubuntu 14.04

**Comment by TheBB** [comment]
As it says, Emacs was built with an older version of libpng than your system is supplying.

Spacemacs can't help you here, you need to fix your Emacs build.

**Comment by TheBB** [comment]
You should probably find out how you installed libpng 1.6 and remove it. The official libpng packages for Ubuntu 14.04 are libpng 1.2.

**Comment by andrewsanchez** [comment]
@TheBB my issue with that is that I installed libpng 1.6 using [these instructions](http://www.linuxfromscratch.org/blfs/view/svn/general/libpng.html) and therefore cannot remove it with my package manager.  I tried downloading it again, moving to the directory created by unpacking [libpng-1.6.21.tar.xz](http://downloads.sourceforge.net/libpng/libpng-1.6.21.tar.xz) and running `make uninstall`, but got the message `make: *** No rule to make target `uninstall'.  Stop.`  I'm not sure if that was the proper way of going about trying to uninstall it.
